### PR TITLE
Release v1.1.0-rc.3

### PR DESCRIPTION
**Pull request by AkihiroSuda**
## Changes (v1.0.2→v1.1.0-rc.1)
See https://github.com/opencontainers/runtime-spec/pull/1175

## Changes (v1.1.0-rc.1→v1.1.0-rc.2)
See https://github.com/opencontainers/runtime-spec/pull/1192

### Additions:

* config: add scheduler entity (#1188)
* config: Add I/O Priority Configuration for process group in Linux Containers (#1191)

### Minor fixes and documentation:

* config-linux: clarify I/O throttling differences between cgroup v1 and v2 (#1194)
* config-linux: chore: Update `ociVersion` in example (#1199)
* releases: use +dev as in-development suffix (#1198)
* MAINTAINERS: add Toru Komatsu (utam0k) (#1201)
* glossary: `s/features document/Features structure/g` (#1203)
* features: update Example (#1204)
* schema: fix definition for ioPriority (#1206)
* CODEOWNER: Add Toru Komatsu(@utam0k) to sync with MAINTAINERS (#1207)

- - -

[[runtime-spec VOTE] tag ae35e39 as v1.1.0-rc3 (closes Mon May 29 02:08:08 PM UTC 2023)](https://groups.google.com/a/opencontainers.org/g/dev/c/VoNKGQpXh70) (at least **8** of 12 maintainers needed for this resolution to pass):
- [ ] @crosbymichael
- [ ] @mrunalp
- [x] @vbatts
- [ ] @dqminh
- [ ] @tianon
- [x] @hqhq
- [x] @cyphar
- [x] @giuseppe
- [X] @AkihiroSuda
- [x] @kolyshkin
- [x] @thaJeztah
- [x] @utam0k

### CODE REVIEW

**Review by utam0k — APPROVED** [review]
🚀

**Review by hqhq — APPROVED** [review]
LGTM

**Review by vbatts — APPROVED** [review]
LGTM

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by cyphar — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
@opencontainers/runtime-spec-maintainers 
We need one more LGTM 🙏 

If you are no longer interested in voting, please let us know.

**Comment by utam0k** [comment]
👋 Hi, @crosbymichael @mrunalp @dqminh @tianon @kolyshkin
You have the last right to vote to release a new version of the OCI Runtime Spec. Please vote for it.

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
Released v1.1.0-rc.3: https://github.com/opencontainers/runtime-spec/releases/tag/v1.1.0-rc.3

Hope we can release v1.1.0 GA by the end of this month.

**Pull request merged**
